apps/web/app/(app)/gardens/[chain]/[garden]/[community]/page.tsx

```diff
@@ -0,0 +1,184 @@
+import { commImg, groupFlowers } from "@/assets";
+import React from "react";
+import Image from "next/image";
+import { EthAddress, Statistic, PoolCard, RegisterMember } from "@/components";
+import { initUrqlClient, queryByChain } from "@/providers/urql";
+import { Address } from "viem";
+import {
+  RegistryCommunity,
+  TokenGarden,
+  getCommunityDocument,
+  getCommunityQuery,
+} from "#/subgraph/.graphclient";
+import {
+  CurrencyDollarIcon,
+  RectangleGroupIcon,
+} from "@heroicons/react/24/outline";
+
+const { urqlClient } = initUrqlClient();
+
+export default async function CommunityPage({
+  params: { chain, garden: tokenAddr, community: communityAddr },
+}: {
+  params: { chain: number; garden: string; community: string };
+}) {
+  const { data: result, error: error } = await queryByChain<getCommunityQuery>(
+    urqlClient,
+    chain,
+    getCommunityDocument,
+    { communityAddr: communityAddr, tokenAddr: tokenAddr },
+  );
+
+  let tokenGarden = result?.tokenGarden;
+
+  let {
+    communityName,
+    members,
+    strategies,
+    communityFee,
+    registerStakeAmount,
+    registerToken,
+    protocolFee,
+    covenantIpfsHash,
+  } = result?.registryCommunity as RegistryCommunity;
+
+  const communityStakedTokens =
+    members?.reduce(
+      (acc: bigint, member) => acc + BigInt(member?.stakedTokens),
+      0n,
+    ) ?? 0;
+
+  strategies = strategies ?? [];
+
+  const signalingPools = strategies.filter(
+    (strategy) => strategy.config?.proposalType === "0" && strategy.isEnabled,
+  );
+
+  const fundingPools = strategies.filter(
+    (strategy) => strategy.config?.proposalType === "1" && strategy.isEnabled,
+  );
+
+  const poolsInReview = strategies.filter((strategy) => !strategy.isEnabled);
+
+  let covenant = "";
+
+  if (covenantIpfsHash) {
+    try {
+      const response = await fetch("https://ipfs.io/ipfs/" + covenantIpfsHash);
+      const json = await response.json();
+      covenant = typeof json.covenant === "string" && json.covenant;
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  const activePools = strategies?.filter((strategy) => strategy?.isEnabled);
+
+  return (
+    <div className="page-layout">
+      <header className="section-layout flex flex-row items-center gap-10">
+        <div>
+          <Image
+            src={commImg}
+            alt={`${communityName} community`}
+            className="h-[180px] bg-slate-200"
```

Comment:
suggestion: remove classNames at Image ?

Reply:
removed it, same as the other image.